purchase_security/security/ir.model.access.csv

```diff
@@ -0,0 +1,2 @@
+id,name,model_id:id,group_id:id,perm_read,perm_write,perm_create,perm_unlink
+purchase_security.access_purchase_team,access_purchase_team,purchase_security.model_purchase_team,base.group_user,1,1,1,1
```

Comment:
This permission is too broad. That's a bad practice. If this is for purchase, why everything is able to modify it? Put read permission for purchase users, and full access to purchase admins.